certified-connectors/YakChat/apiDefinition.swagger.json

```diff
@@ -0,0 +1,382 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "YakChat",
+    "description": "Connector to send and receive SMS and MMS using your YakChat numbers. YakChat adds SMS and MMS to Microsoft Teams so that you can connect instantly with your customers and co-workers using text messaging.",
+    "version": "1.0",
+	"contact": {
+		"name": "YakChat Ltd.",

```

Comment:
@aMatthewsYakChat Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
This has been updated